common/font.go

```diff
@@ -248,6 +248,11 @@ func (f *Font) generateFontAtlas(c int) FontAtlas {
 	return atlas
 }
 
+// Should only use this if you are using your own custom text shader.
```

Comment:
The comment should start with the function name and describe what it does. Something like "GenerateFontAtlas creates an atlas using the Font of the first c UTF-8 characters. This function should only be used in custom Render systems or Shaders. To create the font atlas for the normal RenderSystem, use Create or CreatePreloaded."